mappings/net/minecraft/world/storage/RegionFile.mapping

```diff
@@ -11,12 +11,12 @@ CLASS net/minecraft/class_2861 net/minecraft/world/storage/RegionFile
 	METHOD <init> (Ljava/io/File;Ljava/io/File;Z)V
 		ARG 1 file
 		ARG 2 directory
-		ARG 3 dsync
+		ARG 3 desync
```

Comment:
This is not `desync`. It actually *synchronizes* the data to the **d**isk as soon as it's written, by passing the `StandardOpenOption.DSYNC` option when opening the file. See this discussion when it was added:

https://github.com/FabricMC/yarn/pull/1253#discussion_r402636940

Reply:
Guess we will fix this in some other pr later